source/docs/articles/sites/domains/setting-up-a-domain-with-godaddy.md

```diff
@@ -1,44 +1,27 @@
-
After you complete the purchase process you can begin configuring your domain and you will be presented with the order summary. Click the **My Account** button that will take you to your global GoDaddy Account page.
- ![Sign in to your Godaddy account](/source/docs/assets/images/desk_images/49498.png)
-#### Launch the Domain Control Panel
-
Now you can get to the next stage and click on the **Launch** button which will give you access to your GoDaddy domain control panel. We have instructions if you are using either the [Old](#old-dns-dashboard) or the [New](#new-dns-dashboard) versions of the DNS manager.  
- ![Launch your control panel](/source/docs/assets/images/desk_images/49499.png)  
-### Old DNS Dashboard
-Click the name of the domain that you want to edit and this will take you to the domain manager.  
- ![](/source/docs/assets/images/desk_images/49521.png)  
-Next, you will see the Zone summary screen that will give you the information associated with this domain. Click on the **Launch** button to be taken to the GoDaddy Zone config tool.  
- ![Edit the DNS information](/source/docs/assets/images/desk_images/49502.png)
-### New DNS Dashboard
-
If you are using the New Standard DNS dashboard, you can click on the **Edit Zone** link which will take you directly to the GoDaddy Zone configuration tool.  
- ![DNS dashboard](/source/docs/assets/images/desk_images/49520.png)
-#### Setting Up the Zone Record
-The last step on GoDaddy is to make sure you have correctly setup your A record or CNAME for your domain. Make sure you have put in the I.P. address or the CNAME to point your domain to Pantheon. You can get more information about this on our [Going Live](/docs/articles/going-live/) documentation page. 

+
+Visit GoDaddy.com to search for the domain name you want and add it to your cart. If you have more than one domain, you will be required to configure the A records or CNAME for each one.
+
+If you need to add email to your new domain, you can add it at this time. Pantheon does not provide email hosting services.
+
+After you've purchased your domain name, follow the instructions below.  
+
+## Configure Domain Settings
+
1. Sign in to your account at GoDaddy.com.
+2. Click **Manage My Domains**, then **Manage DNS**.
+3. Enter the **A record** or **CNAME** for your domain. Make sure to enter the IP address or the CNAME to point your domain to Pantheon. For more information, see [Going Live](/docs/articles/going-live/). 
+
 <div class="alert alert-warning" role="alert">
-<strong>Note</strong>: If you have SSL enabled on your site, you have been given a unique I.P. address. This is the I.P. that you should point the A record to.</div>  
- ![Configure the A records](/source/docs/assets/images/desk_images/49503.png)  
-### Add Your Domain to Pantheon
+<strong>Note</strong>: If you have SSL enabled on your site, you have been given a unique IP address. Point the A record to this IP address.</div>  
+
+## Add Your Domain to Pantheon
 
```

Comment:
Adding the domain to Pantheon should be done first - to generate the recommended DNS records that will need to be added at GoDaddy.